### PR TITLE
Clean-up text processing

### PR DESCRIPTION
**Pull request by lizgzil**
Lots of changes are to do with:
- Rename 'useful_functions' -> 'utils'
- Rename 'remove_useless_string -> 'clean_string'
- Move clean_grants_data to utils

I also:
- Modify clean_string to also include converting all the options for 'not available' etc to a unifying label
- Add stopword removal for sklearn vectorizers

### CODE REVIEW

File: nutrition_labels/utils.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import re
+import pandas as pd
+from bs4 import BeautifulSoup
+from sklearn.metrics import confusion_matrix
+
+# Common synopsis/description fields which mean 'No data given'
+not_available_aliases = {
+        'Not available': None,
```
Comment (aCampello) [inline nutrition_labels/utils.py:8]: Are you going to switch to our internal data labs one at some point?

**Review by aCampello — APPROVED** [review]

File: nutrition_labels/utils.py (hunk 1 above)
Comment (lizgzil) [reply nutrition_labels/utils.py:8]: yep :) Will add an issue so it's not forgotten
